Changelog.md

```diff
@@ -12,6 +12,7 @@ Bugfixes:
  * General: Fix internal compiler error when requesting IR AST outputs for interfaces and abstract contracts.
  * SMTChecker: Fix SMT logic error when initializing a fixed-sized-bytes array using string literals.
  * SMTChecker: Fix SMT logic error when translating invariants involving array store and select operations.
+ * SMTChecker: Fix wrong encoding of string literals as arguments of cryptographic functions.
```

Comment:
I think only `ecrecover()` was actually affected.

```suggestion
 * SMTChecker: Fix wrong encoding of string literals as arguments of ``ecrecover()`` precompile.
```

Looking at `symbolicTypeConversion()`, string literal to fixed bytes conversion and conversions to UDVTs are only cases where anything is done.

Reply:
You are right, I update the changelog entry.